src/Core/RevEng.Core.80/ReverseEngineerRunner.cs

```diff
@@ -338,6 +338,26 @@ private static List<string> MoveConfigurationFiles(IList<string> files)
             var movedFiles = new List<string>();
             foreach (var configurationFile in configurationFiles)
             {
+                // This handles the scenario where an Entity ends with the word "Configuration" such as "SoftwareConfiguration".  In this scenario
+                // there would be 2 files "SoftwareConfiguration.cs" and "SoftwareConfigurationConfiguration.cs".  The former is the entity model itself
+                // which should not be moved.  We're also checking to make sure the line isn't part of a comment block. (to handle a scenario where someone
+                // has customized the EntityType.t4 to include a comment block that points to the EntityTypeConfiguration file)
+                bool isConfigFile = false;
+                foreach (var line in File.ReadLines(configurationFile))
```

Comment:
Would it be possible to only do this check based on the end word of the file?

Reply:
How so?  We're already only doing this check if it ends with Configuration.cs.  

Or do you mean something else that my 7AM brain missed? :)

Reply:
Sorry! My mistake...